src/components/form/form.scss

```diff
@@ -10,12 +10,116 @@
     }
 }
 
+.limel-form-layout--default {
+    --gap: 1rem;
+    display: grid;
+    gap: var(--gap);
+}
+
 .limel-form-layout--grid {
+    --gap: 1rem;
+    --min-height-of-one-row: 4.71rem;
     display: grid;
-    grid-template-columns: repeat(auto-fill, minmax(18.75rem, 1fr));
-    gap: 0.5rem;
+    padding: var(--gap);
```

Comment:
This padding is going to be a headache:

![image](https://user-images.githubusercontent.com/154725/101157646-bbaaee80-362a-11eb-8fbe-5094b0a9bbd0.png)

Reply:
This example shows the problem perfectly:

![image](https://user-images.githubusercontent.com/154725/101158282-c7e37b80-362b-11eb-9791-75c531d0705d.png)

The left side of the text and the left side of the first column doesn't line up, and making them line up while this padding is still here is going to be a _lot_ messier than just letting the consumer add the necessary whitespace, when necessary.

If we think this specific amount of padding will often be useful, we can add an optional helper-class that adds the padding.

Reply:
I agree. In a dream world, consumers should put their own padding. But I bet that they don't, because this is a pure UI-design questions; and they don't/can't care about these stuff.

Can we use a var with a default-and-editable value and a fallback (like we do in many other components)? Do you know what I mean? We do things like this: `background-color: var(--icon-background-color, transparent);` in many places. 

Can we do like: `padding: var(--gap, 1rem);`

Reply:
Yeah, ok, that would work, yes. Good idea!

Reply:
Hmm… by the way… is it intentional that this padding is only applied to `.limel-form-layout--grid`, not to `.limel-form-layout--default`? Or is it just an oversight?

Reply:
AFK discussion sum-up: We'll remove the padding for now, so we can get this merged. We can always add it later.